target/linux/ipq40xx/files-6.1/arch/arm/boot/dts/qcom-ipq4019-whw03v2.dts

```diff
@@ -291,7 +291,7 @@
 		led_blue: blue@2 {
 			color = <LED_COLOR_ID_BLUE>;
 			function = LED_FUNCTION_INDICATOR;
-			linux,default-trigger = "default-on";
+			linux,default-trigger = "none";
```

Comment:
You can just remove the trigger completely, that will just leave it at whatever state it was before or if you want to turn it off by default better use the default-off trigger

Reply:
@robimarko thanks for the help!

my testing shows that `none` and not including a `default-trigger` results in exactly the same behavior:
- the bootloader turns on the blue led,
- then openwrt **turns off the led for a second** while bringing up the device,
- just before it flashes it fast in blue showing the failsafe window.

(in code i remember seeing something like a call to reset leds to their default state. it seems this call will react in the same way with `default-trigger` set to `none` or missing: turn the leds off.)

in contrast, when `default-on` is used (current port state), openwrt does not turn off the blue led for a second before the failsafe flash. but then it is kept on the whole time, even after fully booting when the router wants to glow green (kinda mimicking stock behavior), resulting in cyan light.

**Q1:** regarding `default-trigger`, `none` or missing work fine. `default-off` is not needed. would you prefer me to leave the 3 RGB channels as `none` or remove the 3 `default-trigger`s?

Reply:
in stock, green glow does not mean "fully booted", it means "there is internet service". for this port, i would prefer to just leave it blue all the time instead of green. why?
- the ports for EA8300 and MR8300 (the almost-equal-inside bros of the WHW03 routers) chose to leave the led blue, because there is no internet detection, which i think is the right thing to do.
- it frees the green led for custom indication.
- the current 23.x.x release for this router glows blue all the time, not green. (because there is a bug that makes the firmware unable to use the leds at all, which is fixed in master.) so glowing green in the next release would entail an unexpected change of behavior for this device.

**Q2:** would you like me to change the `running` color for this device to blue? this would keep it in line with its current release behavior, the behavior of EA8300 and MR8300, and since i'm currently porting WHW03 V1, i can replicate the behavior there too.

**Q3:** should the eventual color change be different commit?

Reply:
**Q1:** `linux,default-trigger = "none"` is basically equal to not defining one at all, so feel free to drop them.

**Q2:** Yes, changing the current behaviour to be as close to stock FW as possible is encouraged.

**Q3:** Yes, it should be a separate commit with an explanation.

Reply:
@robimarko 

thanks for your help!

i removed the default-triggers and created a separate PR for the color change here:
https://github.com/openwrt/openwrt/pull/15157